src/plugin/action/join.sjs

```diff
@@ -11,13 +11,25 @@ module.exports = function (client, rawf, emitter) {
                 reject(new Error("No channel given to join action."));
                 return;
             }
+            var info = channel;
+            if (typeof(channel) === 'object'){
+                var joinInfo = {
```

Comment:
Don't use `var` more than once in a function. Hoist the `var joinInfo;` to outside of the `if`.